package.json

```diff
@@ -182,7 +182,7 @@
     "**/workbox-build/rollup": "^4.9.1"
   },
   "volta": {
-    "node": "18.18.1",
+    "node": "20.10.0",
     "yarn": "1.22.19"
```

Comment:
I think this version is also different in the new base image.

Reply:
<img width="264" alt="Screenshot 2024-01-03 at 09 52 29" src="https://github.com/porsche-design-system/porsche-design-system/assets/30120450/945c5f97-7370-4787-9d44-a951d8321305">
Seems good to me.